index.js

```diff
@@ -0,0 +1,40 @@
+const exec = require('@actions/exec');
+const core = require('@actions/core');
+const env = process.env;
+
+async function run() {
+    const token = core.getInput('TOKEN');
+    var mainFilePath = core.getInput('MAIN_FILE_PATH')
+
+    if (!mainFilePath) {
+        mainFilePath = './index.js'
+    }
+
+    exec.exec('npm install');
+    exec.exec('npm i -g @vercel/ncc');
```

Comment:
Do we need the `-g`? Can't we do a local install?

Reply:
Not required. We can do local install.